packages/itwin/map-layers/src/ui/FeatureInfoUiItemsProvider.tsx

```diff
@@ -17,15 +17,40 @@ import {
   WidgetState,
 } from "@itwin/appui-react";
 import { IModelApp } from "@itwin/core-frontend";
-import type { MapLayerImageryProvider, ScreenViewport } from "@itwin/core-frontend";
+import type { ScreenViewport } from "@itwin/core-frontend";
+import type { MapLayerProps } from "@itwin/core-common";
 import { SvgMapInfo } from "@itwin/itwinui-icons-react";
 import { MapFeatureInfoTool } from "@itwin/map-layers-formats";
 
 import { MapLayersUI } from "../mapLayers";
 import { MapFeatureInfoWidget } from "./widget/FeatureInfoWidget";
 import type { MapFeatureInfoOptions } from "./Interfaces";
 import { MapLayersSyncUiEventId } from "../MapLayersActionIds";
-import type { MapImagerySettings } from "@itwin/core-common";
+
+const supportsMapFeatureInfo = (vp: ScreenViewport, mapLayerProps: MapLayerProps[]): boolean => {
+  for (let mapLayerIndex = 0; mapLayerIndex < mapLayerProps.length; mapLayerIndex++) {
+    const bgLayerProvider = vp.getMapLayerImageryProvider({ index: mapLayerIndex, isOverlay: false });
```

Comment:
`isOverlay` should be a parameter here, and not hardcoded to false?

Reply:
Yes I overlooked that during the rework.